code/modules/antagonists/horror/horror.dm

```diff
@@ -0,0 +1,836 @@
+/mob/living/simple_animal/horror
+	name = "eldritch horror"
+	real_name = "eldritch horror"
+	desc = "Your eyes can barely comprehend what they're looking at."
+	icon_state = "horror"
+	icon_living = "horror"
+	icon_dead = "horror_dead"
+	icon_gib = "horror_gib"
+	health = 50
+	maxHealth = 50
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	see_in_dark = 7
+	stop_automated_movement = TRUE
+	attacktext = "bites"
+	speak_emote = list("gurgles")
+	attack_sound = 'sound/weapons/bite.ogg'
+	pass_flags = PASSTABLE | PASSMOB
+	mob_size = MOB_SIZE_SMALL
+	faction = list("neutral","silicon","hostile","creature","heretics")
+	ventcrawler = VENTCRAWLER_ALWAYS
+	initial_language_holder = /datum/language_holder/universal
+	hud_type = /datum/hud/chemical_counter
+
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+
+	var/playstyle_string = span_bold(span_big("You are an eldritch horror,") + " an evermutating parasitic abomination. Seek human souls to consume. \
+							Crawl into people's heads and steal their essence. Use it to mutate yourself, giving you access to more power and abilities. \
+							You operate on chemicals that get built up while you spend time in someone's head. You are weak when outside, play carefully.")
+
+	var/mob/living/carbon/victim
+	var/datum/mind/target
+	var/mob/living/captive_brain/host_brain
+	var/truename = null
+	var/available_points = 4
+	var/consumed_souls = 0
+	var/list/horrorabilities = list() //An associative list ("id" = ability datum) containing the abilities the horror has
+	var/list/horrorupgrades = list()		  //same, but for permanent upgrades
+	var/bonding = FALSE
+	var/controlling = FALSE
+	var/chemicals = 10
+	var/chem_regen_rate = 2
+	var/used_freeze
+	var/used_target
+	var/horror_chems = list(/datum/horror_chem/epinephrine,/datum/horror_chem/mannitol,/datum/horror_chem/bicaridine,/datum/horror_chem/kelotane,/datum/horror_chem/charcoal)
+
+	var/leaving = FALSE
+	var/hiding = FALSE
+	var/invisible = FALSE
+	var/waketimerid = null
+	var/datum/action/innate/horror/talk_to_horror/talk_to_horror_action = new
+
+/mob/living/simple_animal/horror/Initialize(mapload, gen=1)
+	..()
+	real_name = "Eldritch horror"
+	truename = "[pick(GLOB.horror_names)]"
+
+	//default abilities
+	add_ability("mutate")

```

Comment:
please make them use paths or something that isn't just strings
Problem is it compiles fine if you make a typo, aka it's prone to introduce annoying bugs

Reply:
You mean i make a bunch of defines for every listed ability? I did it like this, because that's exactly how darkspawn works.

Reply:
Make it work based on typepaths then. Probably not defines, but don’t make it strings

Reply:
Done